<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 34, 34);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>931</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView
{
font: 900 11.5pt &quot;Segoe UI Variable Text Light&quot;  rgb(255, 201, 8);
color: rgb(204, 175, 58)
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>-10</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font:23pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLabel:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>SIMPLE STUDENT INFORMATION SYSTEM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>590</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Delete Students</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>971</x>
      <y>2</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>971</x>
      <y>68</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>130</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Middle Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SurName">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>30</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstName">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>90</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MiddleName">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>150</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>190</y>
      <width>77</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>ID Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDNumber">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>210</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="YearLevel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>270</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>250</y>
      <width>79</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Year Level</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Gender">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>330</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>310</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CourseCode_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>390</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>370</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Course</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>590</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Load Students</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>630</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Load Courses</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_Students">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>430</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Add Student</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>460</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Course Code</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CourseCode">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>490</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}</string>
    </property>
    <property name="text">
     <string>Course Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CourseName">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>560</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_Courses">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>600</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Add Course</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>630</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Delete Courses</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>590</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Edit Student</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>801</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
font: 900 16pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QLineEdit:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Search Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchcourses">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Search Courses</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>630</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Edit Course</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Updatestudent">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>430</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Update Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Updatecourses">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>600</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
font: 900 8pt &quot;MS Reference Sans Serif&quot;rgb(255, 201, 8);
color: rgb(255, 170, 32);
}
QPushButton:Hover
{
color:  rgb(35, 35, 35);
background-color: rgb(220, 182, 44);
}</string>
    </property>
    <property name="text">
     <string>Update Course</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1272</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
